Makefile

```
@@ -367,3 +367,10 @@ clean-mailserver-systemd: ##@Easy Clean your systemd service for running a mails
 
 clean-mailserver-docker: ##@Easy Clean your Docker container running a mailserver
 	@cd _assets/compose/mailserver && $(MAKE) clean
+
+DEFAULT_MIGRATION_PATH := appdatabase/migrations/sql/
+UNIX_TIMESTAMP := $(shell date +%s)
+migration:
```

If the variables are specific to the `make` target, make them actually specific to the target:
```suggestion
migration: DEFAULT_MIGRATION_PATH := appdatabase/migrations/sql/
migration: UNIX_TIMESTAMP := $(shell date +%s)
migration:
```
https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

---

Done